style.css

```diff
@@ -95,16 +119,247 @@ body {
 }
 
 .about-my-self p {
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-size: 16px;
   line-height: 24px;
-  font-weight: 400;
+  font-weight: normal;
   color: #344563;
   margin: 12px 0;
 }
 
 .about-my-self h4 {
-  height: 14px;
+  height: 24px;
+  font-size: 16px;
   font-weight: 500;
   line-height: 24px;
   color: #7f8cff;
   margin: 12px 0;
 }
+
+/**************** work section css start ************/
+
+.button {
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  padding: 5px 8px;
+  text-align: center;
+  color: #396df2;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  margin-top: 15px;
+}
+
+button:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+button:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+button:disabled {
+  color: #5e6c84;
+  background-color: #e5e5e5;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  background-color: #e5e5e5;
+  grid-gap: 88px;
+}
```

Comment:
- [ ] **GOOD JOB SO FAR 🎉😄!**. Here are some bugs to fix but I am sure you can do it 💪. In **works** section cards are not centered and I give suggestion to fix this. Please set justify items to center and done 👌
```css
    justify-items: center;
```

Your design
![image](https://user-images.githubusercontent.com/69011963/148980464-f137726d-235b-48bd-852c-9ca6d8e6051b.png)

Reply:
Ok I will do that thanks a lot for suggestion.